frame/contracts/src/benchmarking/mod.rs

```diff
@@ -2285,7 +2446,7 @@ benchmarks! {
 	// configured `Schedule` during benchmark development.
 	// It can be outputed using the following command:
 	// cargo run --manifest-path=bin/node/cli/Cargo.toml --release \
-	//     --features runtime-benchmarks -- benchmark --dev --execution=native \
+	//     --features runtime-benchmarks -- benchmark --extra --dev --execution=native \
```

Comment:
What's the `extra` feature used for?

Reply:
This was a change introduced to the benchmarking CLI. In order to run benchmarks attributed  with`#[extra]` you need to supply this flag.